SPECS/qemu/qemu.spec

```diff
@@ -1201,7 +1249,6 @@ mkdir -p %{qemu_kvm_build}
   --disable-hax                    \\\
   --disable-hvf                    \\\
   --disable-iconv                  \\\
-  --disable-jemalloc               \\\
```

Comment:
Out of curiosity: is this and the other option below obsolete with the current version of `qemu`?

Reply:
No, it appears to be available still. Just following the lead of the Fedora 36 package here. If we want to keep it on, I don't expect it would cause any issues.

Reply:
Or I should say, if we want to explicitly disable those mallocs

Reply:
I have no preference here, I was just curious what triggered the change.